assets/model_monitoring/components/model_monitor/model_monitor_metric_publisher/spec.yaml

```diff
@@ -0,0 +1,62 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_monitor_metric_outputter
```

Comment:
should this be model_monitor_metric_publisher

Reply:
Fixed.